src/app/components/MessageBanner/index.test.tsx

```diff
@@ -109,5 +110,47 @@ describe('MessageBanner', () => {
       const image = screen.getByAltText('');
       expect(image).toBeInTheDocument();
     });
+    describe('Presence on live environment', () => {
```

Comment:
We're going to fix this up (as it looks like it might have caused snapshot tests to fail for some reason)